src/utils_GUI/GUI_utils.jl

```diff
@@ -204,7 +204,8 @@ end
 For all plotable objects, initialize the available data menu with items.
 """
 function initialize_available_data!(gui)
-    system = get_root_design(gui).system
+    design = get_root_design(gui)
+    system = get_system(design)
```

Comment:
You can, if you do not need design, also have a function `get_system(gui)` which combines the functions.

Reply:
I do need design later on in this function.